include/deal.II/base/parameter_handler.h

```diff
@@ -1503,6 +1505,23 @@ class ParameterHandler : public Subscriptor
   bool
   operator==(const ParameterHandler &prm2) const;
 
+  /**
+   * Returns a set of names corresponding to those entries of the parameter
```

Comment:
```suggestion
   * Return a set of parameter names corresponding to those entries of the parameter
```
Do these names also include subsection names?